src/common/constants.js

```diff
@@ -60,11 +60,12 @@ export const p2p_playstore_url = 'https://play.google.com/store/apps/details?id=
 export const pushwoosh_app_code = 'DD293-35A19'
 export const sample_rate = 25
 export const zoho_url = 'https://deriv.zohorecruit.com'
+export const derivx_url = 'https://deriv.com/derivx/'
```

Comment:
We can user relative url in the component instead of this.

Reply:
please take note of the domain as well 🙏  `${domain.name}`, can be either (.com|.me)